src-tauri/build.rs

```diff
@@ -66,3 +65,37 @@ pub async fn setup_binary() -> Result<()> {
 
     Ok(())
 }
+
+fn determine_target() -> String {
+    // Check if --target flag is provided
+    if let Ok(target) = env::var("CARGO_BUILD_TARGET") {
```

Comment:
I think this should be `TARGET` instead? - https://arc.net/l/quote/oeorzdcc

and it will always be available, not only when `--target` is specified.